### PR TITLE
rebuild with libxml2 2.10 ci skip

### PR DESCRIPTION
**Pull request by akabanovs**
- libxml2 pinned to 2.10
 - includes updates to 'about' section
 - build number increased
 - urls updated
 - build.sh updated
 - patches added

### CODE REVIEW

**Comment by akabanovs** [comment]
ci skip is added to the title of the PR not to trigger prefect build. The reason being:

This package (qt-webengine) has been manually built for linux-aarch64 and linux-64 architectures.  We need to rebuild this package against libxml2 2.10.3 only for these two archs since libxml2 is not used by qt-webengine on other platforms. The package was built manually not to block out the prefect (the package builds more than 5 hours). Moreover, it currently would not build on prefect. The package requires libxkbfile library, not available in the defaults channel. To build it, I installed libxkbfile manually on the dev instance which I used to build the package on. Alternative would be to update the build docker file to have this library available on the build machine and then use prefect, but then it would block the prefect for almost the whole day...

**Review by cbouss — APPROVED** [review]

**Comment by akabanovs** [comment]
oh yes, and some changes are related to osx, even thought we are not going to rebuild/upload for osx (but I was able to build it locally on osx-arm64, but did not try osx-64)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,21 +11,21 @@ source:
...
-  #  git_rev: {{ webengine_version }}  # [not win]
     folder: qtwebengine
     patches:
       - patches/webengine/qt_webengine-3.15.9.patch  # [not win]
       - patches/webengine/0003-win.patch             # [win]
       - patches/webengine/0004-win8.patch            # [win]
+      - patches/webengine/qt_remove_override.patch
 
-  - git_url: https://code.qt.io/qt/qtwebengine-chromium.git  # [not win]
+  - git_url: https://github.com/qt/qtwebengine-chromium.git # [not win]
```
Comment (sumit0190) [inline recipe/meta.yaml]: Is there a reason why we are switching to the GitHub URL? This is a mirror, we would feel more comfortable referencing the original source trees, if possible. It doesn't matter for this particular release, but I just wanted to know if there's a reason why this change was made.

**Review by sumit0190 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: You are right. Both urls in the above work but it is better to use the original source. This change propagated partly by mistake as I was doing some investigation to identify why chromium was giving me build-time problems (eventually I added qtweb_chromium_add_tag.patch to sort it out). I will change the url.
